src/index.js

```diff
@@ -101,6 +101,19 @@ const rule = (properties, options) =>
           }
         }
 
+        if (context.fix && !disableFix) {
+          // only if a fix code is available from config
+          if (options.autofixFunc[property + "-fix"]) {
+            //a is a local variable inside the fix function
+            var a = node.value;
+            // take the fix function from the config
+            node.value = eval(options.autofixFunc[property + "-fix"]);
+            return;
+          } else {
+            console.log('Please include the fix function in your stylelint config file.');
```

Comment:
We cannot use util.report as it is meant for stylelint's report. I am considering to have the error reported from the `validation.js` file. But thats the last option because even from there we cannot give a descriptive message to the consumer.

Reply:
Updated: removed the `else` block. I forgot that validation.js already takes care of it.